dRAGon/search.py

```diff
@@ -34,7 +40,7 @@ def search_rulebook(query_text=""):
 # create index if it does not exist
 if not os.path.exists("indexdir"):
     # Load data
-    with open("../../cairn-markdown.md") as file:
+    with open("../../"+config['rulebook']) as file:
```

Comment:
In the future we should fix this to use "real" python relative pathing rather than ../../ which may fail in cases where run from a different directory